packages/accordions/src/elements/accordion/components/Header.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/* 
+  Keyboard visual indicators are separate from hover visual indicators as outlined by the 
+  `Accordion.Header` design, so there are no key events associated to mouse events. Ignoring this
+  lint rule is uncommon, but required for Garden's `Accordion.Header`.
+*/
+
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+
+import React, { useState, FocusEvent, forwardRef, HTMLAttributes } from 'react';
+import { composeEventHandlers } from '@zendeskgarden/container-utilities';
+import ChevronDown from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
+import { useAccordionContext, useSectionContext, HeaderContext } from '../../../utils';
+import { StyledHeader, StyledRotateIcon, COMPONENT_ID as buttonGardenId } from '../../../styled';
+
+export interface IHeaderProps extends HTMLAttributes<HTMLDivElement> {
+  /** Apply props to the wrapping `Accordion.Header` element */
+  wrapperProps?: any;
```

Comment:
Why is it necessary to pass `wrapperProps` and use context with the current separation?

Can we have a standard DOM mapping of:
- `Accordion.Header` - `getHeaderProps()`
- `Accordion.Label` - `getTriggerProps()`

Reply:
> Why is it necessary to pass `wrapperProps`

Passing `wrapperProps` allows consumers to style the wrapping heading element. Do you think `wrapperProps` should be passed to the button element instead? Or to eliminate this prop completely?

> Can we have a standard DOM mapping of:

The current code is similar to this mapping, except the `onClick` from `getTriggerProps` is plucked off, and passed along with the `getHeaderProps` object. This is done to allow clicking the header to trigger expanding / collapsing the accordion sections. Is there a better alternative to achieve this behavior?

Reply:
Updated 🎉 .